src/github/index.ts

```diff
@@ -25,11 +25,12 @@ import {
   GitHubConflictError,
   isGitHubError,
 } from './common/errors.js';
+import { VERSION } from "./common/version.js";
 
 const server = new Server(
   {
     name: "github-mcp-server",
-    version: "0.1.0",
```

Comment:
I'm replacing this value with the contents of `package.json`, but I have also noticed that npm has a totally different package version. What should we use here?

Reply:
I've proposed a CI check here: https://github.com/modelcontextprotocol/servers/pull/593